ios/Classes/utils/ConfigurationMapper.swift

```diff
@@ -107,8 +107,7 @@ extension DropInConfigurationDTO {
 
 extension CardConfigurationDTO {
     func mapToCardComponentConfiguration(shopperLocale: String?) -> CardComponent.Configuration {
-        var formComponentStyle = AdyenAppearanceLoader.findCardComponentStyle() ?? FormComponentStyle()
-        formComponentStyle.backgroundColor = UIColor.white
+        let formComponentStyle = AdyenAppearanceLoader.findCardComponentStyle() ?? createDefaultFormComponentStyle()
```

Comment:
`style` is optional - you can simply provide "as-is"

```suggestion
        let formComponentStyle = AdyenAppearanceLoader.findCardComponentStyle()
```